src/main/kotlin/org/radarbase/redcap/managementportal/Source.kt

```diff
@@ -0,0 +1,33 @@
+package org.radarbase.redcap.managementportal
+
+import java.util.*
+
+
+/*
+ * Copyright 2017 King's College London
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+data class Source(
```

Comment:
do we need to parse all this info? Can we just read it as JsonNode/array/tree and just pass that back?

Reply:
Ah yes `JsonNode` should work. Let me update.